src/app/+admin/admin-registries/metadata-registry/metadata-registry.reducers.ts

```diff
@@ -0,0 +1,106 @@
+import { MetadataSchema } from '../../../core/metadata/metadataschema.model';
+import {
+  MetadataRegistryAction,
+  MetadataRegistryActionTypes,
+  MetadataRegistryDeselectFieldAction,
+  MetadataRegistryDeselectSchemaAction,
+  MetadataRegistryEditFieldAction,
+  MetadataRegistryEditSchemaAction,
+  MetadataRegistrySelectFieldAction,
+  MetadataRegistrySelectSchemaAction
+} from './metadata-registry.actions';
+import { MetadataField } from '../../../core/metadata/metadatafield.model';
+
+/**
+ * The auth state.
+ * @interface State
+ */
+export interface MetadataRegistryState {
+  editSchema: MetadataSchema;
+  selectedSchemas: MetadataSchema[];
+  editField: MetadataField;
+  selectedFields: MetadataField[];
+}
+
+/**
+ * The initial state.
+ */
+const initialState: MetadataRegistryState = {
+  editSchema: null,
+  selectedSchemas: [],
+  editField: null,
+  selectedFields: []
+};
+
+export function metadataRegistryReducer(state = initialState, action: MetadataRegistryAction): MetadataRegistryState {
```

Comment:
Please add TypeDocs to describe this function

Reply:
Added